pytorch_lightning/loops/dataloader/evaluation_loop.py

```diff
@@ -98,7 +99,9 @@ def advance(self, *args: Any, **kwargs: Any) -> None:
         """Performs evaluation on one single dataloader"""
         void(*args, **kwargs)
         dataloader = self.trainer.accelerator.process_dataloader(self.current_dataloader)
-        dataloader_iter = enumerate(dataloader)
+        data_fetcher = DataFetcher()
```

Comment:
the logic to change this to a custom class is yet to come?

Reply:
Yes.

Reply:
For now, this is just used internally.